src/main/java/core/basesyntax/StorageImpl.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+public class StorageImpl<K, V> implements Storage<K, V> {
+    static final int MAX_NUMBER = 10;
+    private final Object[] keyArray = new Object[MAX_NUMBER];
+    private final Object[] valueArray = new Object[MAX_NUMBER];
+
+    @Override
+    public void put(Object key, Object value) {
```

Comment:
why did we need to parametrize a class if you didn't use it in any way?)
```suggestion
    public void put(K key, V value) {
```